src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -95,6 +101,7 @@ export default App.extend({
 
     this.showChildView('list', collectionView);
 
+    this.setState('isSearchDisabled', !collection || !collection.length);
```

Comment:
I'm not sure this is a state attribute that is necessary.  I think `showSearchView` can be called here and it can just use `isDisabled = !this.collection || !this.collection.length`